src/main/webapp/app/exam/manage/student-exams/student-exams.component.ts

```diff
@@ -98,9 +98,9 @@ export class StudentExamsComponent implements OnInit {
     }
 
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    viewAssessment(studentExam: StudentExam) {
-        // TODO: go to assessment
-    }
+    // viewAssessment(studentExam: StudentExam) {
```

Comment:
I would delete these lines instead of commenting them out.

Reply:
yeah, I deleted it.